monolog/monolog.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/*
+ * Name: Monolog
+ * Description: A Logging framework with lots of additions (see [Monolog](https://github.com/Seldaek/monolog/)). There are just Friendica additions inside the src directory
+ * Version: 1.0
+ * Author: Philipp Holzer
+ */
+
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Addon\monolog\src\IntrospectionProcessor;
+use Friendica\DI;
+use Psr\Log\LogLevel;
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php';
```

Comment:
The `DIRECTORY_SEPARATOR` constant isn't meaningful, both Linux and Windows system use `/` to compute paths, and it's shorter to write.

This isn't the case for `PHP_EOL` which should still be used.